scrapy/extensions/feedexport.py

```diff
@@ -550,6 +550,11 @@ def _load_filter(self, feed_options):
         item_filter_class = load_object(feed_options.get("item_filter", ItemFilter))
         return item_filter_class(feed_options)
 
-    def _load_batch(self, feed_options):
-        batch_class = load_object(feed_options.get("batch", BatchHandler))
-        return batch_class(feed_options)
+    def _load_batch(self, feed_options, uri):
+        batch_class = load_object(feed_options.get("batch", DefaultBatchHandler))
+        try:
+            batch_obj = batch_class(feed_options)
+        except NotConfigured:
+            logger.warning("Batch Handler %s not configured for %s." % (batch_class.__name__, uri))
```

Comment:
This should probably be not be a warning, since batch handling not being configured is not only OK, but will also be the case most of the time. I’m not sure we need a message here in the first place, `NotConfigured` means the component controlled the issue. We can leave up to the component to log a warning if it is warranted.